ministryofjustice/analytical-platform-infrastructure
# External DNS

Creating a role that the cluster can assume to modify dns records
on behalf of the **External DNS** pods.

__Generic Role__:
* Added a `generic_role` terraform module to create an IAM role with
attached IAM policy

__External DNS__:
* Invoke `generic_role` module to create role and attached policy for
**External DNS** pods
* Added spec tests

**Deployed**

**To Test**
- Ensure you have your `landing` IAM account profile selected/exported
- Pull branch
- Remove lines `10` and `17` from `spec/spec_helper.rb`
- Run `bundle exec rake spec`
- Should all be green
